umd_rebuild.html

```diff
@@ -34,7 +31,6 @@
         <img src="www.umd.edu/calendar"/>
       </a>
     </section>
-
     <section>
       <h1>In the <strong>News</strong></h1>
         <article>
```

Comment:
This article should be at the same level as the h1 right above. Refactor indentation.